quiz.css

```diff
@@ -0,0 +1,30 @@
+article{
+  width: 33%;
+  border: solid lightblue 1px;
+  display: inline-block;
+}
+
+h1{
+  border: dotted lightgray 1px;
+  text-align: center;
+}
+
+.image{
+  text-align: center;
+}
+
+.specifications div:nth-of-type(n+0){
```

Comment:
Sorry just getting around to this!  Yes, this is what I am using, in lieu of placing a class on every element, I am using it to make all the Price and Specification items BOLD per the instructions.
> On Aug 2, 2016, at 8:42 PM, Nathan Gonzalez <notifications@github.com> wrote:
> 
> In quiz.css <https://github.com/nss-evening-cohort-04/mastery-exercise-html-jkillz2020/pull/1#discussion_r73268685>:
> 
> > +article{
> > +  width: 33%;
> > +  border: solid lightblue 1px;
> > +  display: inline-block;
> > +}
> > +
> > +h1{
> > +  border: dotted lightgray 1px;
> > +  text-align: center;
> > +}
> > +
> > +.image{
> > +  text-align: center;
> > +}
> > +
> > +.specifications div:nth-of-type(n+0){
> What does this pseudo-class do for you here? Is it necessary?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/nss-evening-cohort-04/mastery-exercise-html-jkillz2020/pull/1/files/e7ef6bdd4f5a5ee054805768e28d4e6d74cdd15a#r73268685>, or mute the thread <https://github.com/notifications/unsubscribe-auth/APtwkQBw5K6tO_8RRhZQr-Q-OCxV7DlHks5qb_IdgaJpZM4JVwLN>.
>